Makefile

```diff
@@ -0,0 +1,30 @@
+#
```

Comment:
for easier building steps (read: don't require make) and cross-platform compatibility (ie. to run this locally), can we convert this Makefile into a  shelljs (node) script ?

Reply:
What build system doesn't have `make` installed? besides, `make` is a build system, not a shell, we may replace it with something like `gulp` instead (much more code to write though). Feel free to add a story for making the build process work on Windows :D

Reply:
actually it works on windows (that's my developement environment =)